src/main/java/cc/hyperium/handlers/handlers/keybinds/HyperiumBind.java

```diff
@@ -48,14 +48,7 @@ public void setKey(int key){
 
     public void onPress(){
         if(!pressed){
-            if(!activated){
-                System.out.println("[KEYBINDS] " + name + "keybind enabled!");
-                EventBus.INSTANCE.post(new KeyBindEnableEvent(key));
-            } else{
-                System.out.println("[KEYBINDS] " + name + "keybind disabled!");
-                EventBus.INSTANCE.post(new KeyBindDisableEvent(key));
-            }
-            activated = !activated;
+            EventBus.INSTANCE.post(new KeyBindPressEvent(key));
```

Comment:
What if one wants to detect a key release? Probably should include a variable for whether this is a press or release.